content/hi/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,119 @@
+
+नतीजतन, नोड्स और पॉड्स से कनेक्शन के लिए डिफ़ॉल्ट ऑपरेटिंग मोड चल रहा है
+नियंत्रण विमान के नोड्स डिफ़ॉल्ट रूप से सुरक्षित होते हैं और अविश्वसनीय और/या सार्वजनिक रूप से चल सकते हैं
+नेटवर्क।
+
+## विमान को नोड पर नियंत्रित करें
+
+कंट्रोल प्लेन (एपीआई सर्वर) से नोड्स तक दो प्राथमिक संचार पथ हैं।
+पहला एपीआई सर्वर से क्यूबलेट प्रक्रिया है जो क्लस्टर में प्रत्येक नोड पर चलता है।
+दूसरा एपीआई सर्वर से किसी भी नोड, पॉड या सेवा के लिए एपीआई सर्वर के _proxy_ के माध्यम से है
+कार्यक्षमता।
+
+क्यूबलेट के लिए ### एपीआई सर्वर
+
+एपीआई सर्वर से क्यूबलेट के लिए कनेक्शन का उपयोग इसके लिए किया जाता है:
+
+* पॉड के लिए लॉग लाना।
+* रनिंग पॉड से जोड़ना (आमतौर पर `कुबेक्टल` के माध्यम से)।
+* क्यूबलेट की पोर्ट-फ़ॉरवर्डिंग कार्यक्षमता प्रदान करना।
+
+ये कनेक्शन क्यूबलेट के HTTPS समापन बिंदु पर समाप्त हो जाते हैं। डिफ़ॉल्ट रूप से, एपीआई सर्वर नहीं करता है
+क्यूबलेट के सर्विंग सर्टिफिकेट को सत्यापित करें, जो कनेक्शन को मैन-इन-द-बीच के अधीन बनाता है
+अविश्वसनीय और/या सार्वजनिक नेटवर्क पर चलने के लिए हमले और **असुरक्षित**।
+
+इस कनेक्शन को सत्यापित करने के लिए, एपीआई प्रदान करने के लिए `--क्यूबलेट-सर्टिफिकेट-प्राधिकरण` ध्वज का उपयोग करें
+क्यूबलेट के सर्विंग सर्टिफिकेट को सत्यापित करने के लिए उपयोग करने के लिए रूट सर्टिफिकेट बंडल वाला सर्वर।
+
+यदि यह संभव नहीं है, तो एपीआई सर्वर और क्यूबलेट के बीच [SSH टनलिंग](#ssh-tunnels) का उपयोग करें यदि
+a से कनेक्ट होने से बचने के लिए आवश्यक है
+अविश्वसनीय या सार्वजनिक नेटवर्क।
+
+
+अंत में, [क्यूबलेट प्रमाणीकरण और/या प्राधिकरण](/docs/reference/access-authn-authz/kubelet-authn-authz/)
+क्यूबलेट एपीआई को सुरक्षित करने के लिए सक्षम होना चाहिए।
+
+### एपीआई सर्वर नोड्स, पॉड्स और सेवाओं के लिए
+
+एपीआई सर्वर से नोड, पॉड, या सेवा डिफ़ॉल्ट रूप से सादे HTTP कनेक्शन के लिए कनेक्शन
+और इसलिए न तो प्रमाणित हैं और न ही एन्क्रिप्ट किए गए हैं। उन्हें एक सुरक्षित HTTPS पर चलाया जा सकता है
+API URL में नोड, पॉड, या सेवा के नाम में `https:` प्रीफ़िक्स लगाकर कनेक्शन, लेकिन वे करेंगे
+HTTPS समापन बिंदु द्वारा प्रदान किए गए प्रमाणपत्र को मान्य नहीं करता है और न ही क्लाइंट क्रेडेंशियल्स प्रदान करता है। इसलिए
+जबकि कनेक्शन एन्क्रिप्ट किया जाएगा, यह अखंडता की कोई गारंटी नहीं देगा। इन
+कनेक्शन **वर्तमान में सुरक्षित नहीं हैं** अविश्वसनीय या सार्वजनिक नेटवर्क पर चलाने के लिए।
+
+### SSH tunnels
+
+Kubernetes संचार पथों को नियंत्रित करने के लिए नियंत्रण विमान की सुरक्षा के लिए SSH सुरंगों का समर्थन करता है। इस में
+कॉन्फ़िगरेशन, एपीआई सर्वर क्लस्टर में प्रत्येक नोड के लिए एक एसएसएच सुरंग शुरू करता है (कनेक्ट करने के लिए
+SSH सर्वर पोर्ट 22 पर सुन रहा है) और क्यूबलेट, नोड, पॉड, या के लिए नियत सभी ट्रैफ़िक को पास करता है
+सुरंग के माध्यम से सेवा।
+यह टनल सुनिश्चित करती है कि ट्रैफ़िक उस नेटवर्क के बाहर उजागर न हो जिसमें नोड हैं
+दौड़ना।
+
+{{< नोट >}}
+SSH सुरंगों को वर्तमान में पदावनत कर दिया गया है, इसलिए आपको उनका उपयोग तब तक नहीं करना चाहिए जब तक कि आप यह नहीं जानते कि आप क्या हैं
+कर रहे हैं। [कनेक्टिविटी सेवा](#कनेक्टिविटी-सेवा) इसका प्रतिस्थापन है
+बातचीत का माध्यम।
+{{< /नोट >}}
```

Comment:
Avoid localizing the shortcode name to Hindi, as it results in [errors](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65723c8e06331400086c39fd). Keep the shortcode name as it is and localize the content inside.
```suggestion
{{< note >}}
SSH सुरंगों को वर्तमान में पदावनत कर दिया गया है, इसलिए आपको उनका उपयोग तब तक नहीं करना चाहिए जब तक कि आप यह नहीं जानते कि आप क्या हैं
कर रहे हैं। [कनेक्टिविटी सेवा](#कनेक्टिविटी-सेवा) इसका प्रतिस्थापन है
बातचीत का माध्यम।
{{< /note >}}
```